entries/i/invoiced.com.json

```
@@ -5,7 +5,10 @@
       "sms",
       "custom-software"
     ],
-    "documentation": "https://support.invoiced.com/knowledge_base/topics/setting-up-two-factor-authentication",
+    "custom-software": [
+      "Authy"
+    ],
+    "documentation": "https://docs.invoiced.com/faqs#wb-does-invoiced-support-two-factor-authentication-2fa",
```

The other link doesn't direct me to the right part of the page.
```suggestion
    "documentation": "https://docs.invoiced.com/faqs#wBTZz",
```